src/lxml/html/tests/test_diff.txt

```diff
@@ -87,6 +87,13 @@ Whitespace is generally ignored for the diff but preserved during the diff:
     second
      <ins>third</ins> </pre>
 
+Ensure we don't preserve the structure of the html during the diff:
```

Comment:
"don't" ?

Reply:
Ah sorry, that was a brain frat ! Good catch. I think I might have wanted to add something like 'Ensure we don't mess with the structure ...' and instead decided to go with the opposite phrasing 'Ensue we preserve ...' but obviously messed up. I'll fix in the next commit.

Reply:
Done